File: test/samples/sample.scala
object Sample {
  def classify(a: Int, b: Int): Int = {
    if (a > 0 && b > 0) {
      1
    } else if (a < 0) {
      2
    } else {
      3
    }
  }

  def loop(n: Int, m: Int): Int = {
    var total = 0
    for (i <- 0 until n) {
      while (total < m) {
        if (total == 7) {
          total = total + 2
        }
        total = total + 1
      }
    }
    total
  }
}

File: test/samples/sample.cpp
class Sample {
public:
int classify(int a, int b) {
    if (a > 0 && b > 0) {
        return 1;
    } else if (a < 0) {
        return 2;
    } else {
        return 3;
    }
}
};

int loop(int n, int m) {
    int total = 0;
    for (int i = 0; i < n; i++) {
        while (total < m) {
            if (total == 7) {
                total = total + 2;
            }
            total = total + 1;
        }
    }
    return total;
}
